setup/WDS.nsi

```diff
@@ -267,6 +267,7 @@ Function CreateUninstallEntry
   WriteRegExpandStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\WinDirStat" "InstallLocation" "$INSTDIR"
   WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\WinDirStat" "DisplayName" "${sVersionFull}"
   WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\WinDirStat" "DisplayIcon" "$INSTDIR\windirstat.exe,0"
+  WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\WinDirStat" "DisplayVersion" "${dVersionMajor}.${dVersionMinor}.${dVersionRev}"
```

Comment:
Would be better to use version already formatted as a string, probably `${sVersionS}` or either of `${sVersion_Ansi}` and `${sVersion_Unicode}` if they always point to the same version

Reply:
Nice, didn't know there was a preformatted version string.
As long as it's consistent, it shouldn't really matter I think.

But can change to `${sVersionS}`, I didn't find any references where it's being set, so I opted for the safe route of constructing it manually

Reply:
> Just to be sure I downloaded from fosshub just now, and the filename comes out as `windirstat1_1_2_setup.exe` but maybe the most important part is that the what is injected to the properties of the installer is the full version i.e `<major>.<minor>.<revision/patch>.<build>`

The `1_1_2` string comes from `${sVersionFile}`, see:
https://github.com/windirstat/windirstat/blob/351490639619ee2273a18d05105a1f11fe627341/setup/WDS.nsi#L19
The FileVersion in the properties of the installer (1.1.2.79) comes from `${sVersionS}`, see:
https://github.com/windirstat/windirstat/blob/351490639619ee2273a18d05105a1f11fe627341/setup/WDS.nsi#L86
This for some reason is different from the version in the program's exe (1.1.2.80).


> But can change to `${sVersionS}`, I didn't find any references where it's being set, so I opted for the safe route of constructing it manually

Yeah, those are probably set internally, I agree that without knowing constructing it manually was the best choice.
Maybe @assarbad can say which one should be preferred `${dVersionMajor}.${dVersionMinor}.${dVersionRev}`, `${sVersionS}` or `${sVersion_Ansi}`/`${sVersion_Unicode}`?